### ISSUE TITLE
[Docs] openid-client listed as peerDependency, but it's not

### ISSUE THREAD

**Issue by MonstraG** [nextauthjs/next-auth#6142]
### What is the improvement or update you wish to see?

https://next-auth.js.org/errors says:

> ...it might have happened due to openid-client (which is peer dependency) node version mismatch.

`openid-client` is not a peer dependency according to this:

https://github.com/nextauthjs/next-auth/blob/main/packages/next-auth/package.json#L75

I also tried to click `Edit this page` at the bottom, but that sent me to

https://github.com/nextauthjs/next-auth/edit/main/docs/docs/errors.md

which returned 404.

### Is there any context that might help us understand?

n/a

### Does the docs page already exist? Please link to it.

https://next-auth.js.org/errors

**Comment by iswarmondal** [comment]
Hey @balazsorban44, let me rephrase the words and create a PR accordingly
